src/lib/Event/FormActionEvent.php

```diff
@@ -14,47 +14,25 @@
 
 class FormActionEvent extends FormEvent
 {
-    /**
-     * Name of the button used to submit the form.
-     *
-     * @var string
-     */
-    private $clickedButton;
-
-    /**
-     * Hash of options.
-     *
-     * @var array
-     */
-    private $options;
-
     /**
      * Response to return after form post-processing. Typically a RedirectResponse.
-     *
-     * @var \Symfony\Component\HttpFoundation\Response
-     */
-    private $response;
-
-    /**
-     * Additional payload populated for event listeners next in priority.
-     *
-     * @var array
      */
-    private $payloads;
+    private ?Response $response = null;
 
     /**
      * @param \Symfony\Component\Form\FormInterface $form
      * @param $data
      * @param $clickedButton
      * @param array $options
      * @param array $payloads
+     * @param string $clickedButton
+     */
+    public function __construct(FormInterface $form, mixed $data, /**
```

Comment:
Could those be `readonly` as well? Besides, each parameter should be in a new line.